### ISSUE TITLE
New modules to check into IzzyOnDroid

### ISSUE THREAD

**Issue by binarynoise** [binarynoise/XposedModulets#6]
Hi @izzysoft,
@programminghoch10 and I worked on and added some new modulets.
Would you add the new ones to your repo?

- [x] com.programminghoch10.AntiBrightnessChange
- ~~com.programminghoch10.MotionEventMod~~
- [x] com.programminghoch10.RotationControl
- [x] de.binarynoise.AutomaticAdvancedSettingsExpander
- [x] de.binarynoise.betterBluetoothDeviceSort
- [x] de.binarynoise.betterVerboseWiFiLogging
- ~~de.binarynoise.dontResetIfBootedAndConnected~~
- [x] de.binarynoise.freeNotifications
- ~~de.binarynoise.muteSlf4jWarnings~~
- [ ] de.binarynoise.openWifiOnTop
- [ ] de.binarynoise.persistentForegroundServiceNotifications
- [x] de.binarynoise.resetAllNotificationChannels

---

- [x] added to the repo
- [ ] not added to the repo yet
- ~~not to be added to the repo~~

**Comment by IzzySoft** [comment]
Would be nice if your Fastlane could include an icon for each module. Quick idea: the standard Xposed icon with a symbol embedded, e.g. the standard brightness symbol wtih AntiBrightnessChange, rotation symbol with RotationControl etc. Not the nicest ones admittedly, but if you like them:

![icon_bluetooth](https://github.com/binarynoise/XposedModulets/assets/6781438/3fd9b726-2aeb-4a9b-85fc-3ac31209f835)
![icon_brightness](https://github.com/binarynoise/XposedModulets/assets/6781438/8a95130b-ca1e-4df8-b21f-fca03a043a85)
![icon_rotate](https://github.com/binarynoise/XposedModulets/assets/6781438/7cf178bb-e9a6-4bef-bb70-7bdfd624847a)
![icon_settings](https://github.com/binarynoise/XposedModulets/assets/6781438/c133a879-d720-49bc-b735-3158462d5b27)
![icon_wifi](https://github.com/binarynoise/XposedModulets/assets/6781438/77a966c3-1730-4ec1-9e1b-9a6ab2346e2b)

In case you want to edit them, or build upon it (e.g. for the other modules), here's the multi-layer Gimp file: [xposed_icons.xcf.zip](https://github.com/binarynoise/XposedModulets/files/13801478/xposed_icons.xcf.zip). I have no idea why in the browser, the background of the puzzle piece is gray; it's white with 25% opacity and an image viewer also shows it as such. If you prefer it entirely transparent, just select the other "piece" layer in the Gimp file and export again. Or fill it with a different color, e.g. a decent and not too bright yellow (complement to the blue).

* `com.programminghoch10.RotationControl`: full_description is just a copy of short_description. Can you please update that similar to com.programminghoch10.AntiBrightnessChange?
* `de.binarynoise.muteSlf4jWarnings`: Fastlane directory is empty, need metadata before I can add this :wink:

**Comment by IzzySoft** [comment]
PS: `com.programminghoch10.RotationControl` gives me the following whenever I create a new index here:

```
ResourceType W 12-31 15:19:54 3200440 3200440] No known package when getting value for resource number 0x0108004d
ResourceType W 12-31 15:19:54 3200440 3200440] No known package when getting value for resource number 0x0108004d
AndroidManifest.xml:11: error: ERROR getting 'android:icon' attribute: attribute value reference does not exist
```

(just with different time stamps, which reflect the time `fdroid update` was run). Can you please check and fix that? It's otherwise spamming me constantly :see_no_evil:

**Comment by IzzySoft** [comment]
> I uploaded a version of RotationControl that has the icon removed, did this fix your issue with publishing?

Checking now … `iod repo get com.programminghoch10.RotationControl` successful … `iod repo fdu` running … done. Confirmed, looks good – thanks!

> I will have a look at creating the icons.

Wonderful!

**Comment by BRBsoup** [comment]
Probably would be cool to state that they're xPosed modules in the descriptions :)

**Comment by IzzySoft** [comment]
They are in the "Xposed" category, no?

**Comment by cooldroid** [comment]
@binarynoise @IzzySoft - I tried the ForegroundNotification module on OOS 14 (OnePlus 11) and it doesn't seem to work. Any leads on this please?

**Comment by IzzySoft** [comment]
I cannot tell, sorry – IzzyOnDroid is "just" a distribution channel. So this needs to be answered by @binarynoise …

**Comment by binarynoise** [comment]
> state that they're xPosed modules in the descriptions

Yeah, this might be a good idea. I'll add it with the next update if I don't forget.

**Comment by binarynoise** [comment]
@BRBsoup
I updated the descriptions to mention Xposed.

@IzzySoft 
I added two modulets: OpenWifiOnTop and PersistentForegroundServiceNotifications. Would you be so kind to add them to the repo?
After that, I'd finally close this issue and create separate issues for new modulets.

Yes I know, icons are missing, I'll eventually add some, but I find it difficult to find the time and make some proper ones. #10
